static/app/types/group.tsx

```diff
@@ -715,9 +715,12 @@ export interface BaseGroup {
   participants: Array<UserParticipant | TeamParticipant>;
   permalink: string;
   platform: PlatformKey;
-  pluginActions: any[]; // TODO(ts)
-  pluginContexts: any[]; // TODO(ts)
-  pluginIssues: any[]; // TODO(ts)
+  pluginActions: any[];
+  // TODO(ts)
+  pluginContexts: any[];
+  // TODO(ts)
+  pluginIssues: any[];
+  // TODO(ts)
```

Comment:
todo's

Reply:
pretty sure no types exist yet because everything related to external issues is a mess, but definitely can look into later